index.html

```diff
@@ -23,65 +23,78 @@
         </div>
+        <ul class="nav_items">
+          <li class="nav_item">
+            <a class="nav_item_action" href="#portfolio">Portfolio</a>
+          </li>
+          <li class="nav_item">
+            <a class="nav_item_action" href="#about">About</a>
+          </li>
+          <li class="nav_item">
+            <a class="nav_item_action" href="#contact">Contact</a>
+          </li>
+        </ul>
       </nav>
     </header>
-    <!-- the Main Container -->
+    <!-- Main Container -->
     <main class="main_container">
-
       <!-- Headline section -->
-
       <section class="headline">
-        <h1 class="headline_title">I’m Ibrahim Glad to see you!</h1>
-        <p class="headline_content">
-          I’m a software developer! I can help you build a product , feature or
-          website Look through some of my work and experience! If you like what
-          you see and have a project you need coded, don’t hestiate to contact
-          me.
-        </p>
-        <div class="headline_social">
-          <h3 class="social_text blue">LET’S CONNECT</h3>
-          <ul class="social_links">
-            <li class="social_link">
-              <a href="https://www.twitter.com/kingibro345" target="_blank"
-                ><img
-                  class="social_icon"
-                  src="./images/twitter.svg"
-                  alt="Twitter"
-              /></a>
-            </li>
-            <li class="social_link">
-              <a href="https://www.linkedin.com/ibrahima" target="_blank"
-                ><img
-                  class="social_icon"
-                  src="./images/linkedin-icon.svg"
-                  alt="Linkedin"
-              /></a>
-            </li>
-            <li class="social_link">
-              <a href="https://www.medium.com" target="_blank"
-                ><img class="social_icon" src="./images/m.svg" alt="M"
-              /></a>
-            </li>
-            <li class="social_link">
-              <a href="https://www.github.com/Dmambo"
-                ><img
-                  class="social_icon"
-                  src="./images/github.svg"
-                  alt="Github"
-              /></a>
-            </li>
-            <li class="social_link">
-              <a href="https://www.Diot.com" target="_blank"
-                ><img class="social_icon" src="./images/doit.svg" alt="Diot"
-              /></a>
-            </li>
-          </ul>
+        <div class="headline_container">
+          <h1 class="headline_title">
+            I’m Ibrahim <br />
+            Glad to see you!
+          </h1>
+          <p class="headline_content">
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
```

Comment:
[Optional]

- [ ] Kindly increase the `font-size` of the headline's text to match Figma and to make it look better because right now it's looks too small for desktop 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/199987006-40e4b171-e7e9-4542-b0ad-f4fdcb835eec.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/199987084-7924371c-2ea6-470f-b644-f913843e4df5.png)